basics/answers.md

```diff
@@ -296,6 +305,29 @@ isn't exactly like mine but it works just fine it's ok, and you can
     print(converted_numbers)
     ```
 
+5. ``` python
+   row_count = int(input("Type the number of rows needed:")) 
+   for column_count in range(1, row_count+1):
+       for number in range(1, column_count+1): 
+           print(number, end=" ") 
+       print()  # creates a new line for the next row
+   ```
+   We actually use the first `for` loop to loop through the number of rows needed.Now,
+   through that `for` loop we got to which column the number should be inserted.
+   Now,the next `for` loop helps us to identify which numbers should be printed on the row which we currently accessed from the `for` loop above
+
+   
+  
+6. The idea is more or less same as in the above question.
+   ```python
+   row_count=int(input("Type the number of rows needed:"))
+    
+   for column_count in range(1, row_count + 1): 
+     for number in range(row_count,column_count+1): 
```

Comment:
This code still doesn't make sense to me when I read it. It also doesn't work.

The range must start from row number, which goes from 1 to row count. So how about this?

```suggestion
   for row_number in range(1, row_count + 1):
     for number in range(row_number, row_count+1):
```

It still needs a better explanation. "More or less same as above" is not enough, because we now have to start the range from `row_number`, not from `1`. For example, on row number 2, we want to print numbers from 2 to 5. It is what the range does, but for someone who is new to ranges, that isn't obvious.